src/App.js

```diff
@@ -1,29 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator';
 
-function App() {
+export default function App() {
```

Comment:
* Kindly Use class-based components here as stated in the project [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components.).

Reply:
Hello @Davidosky007 ,

Kindly note: I used a class-based component for the Calculator, as per advised.
Kindly check the Calculator Component.

Reply:
![image](https://user-images.githubusercontent.com/52497246/143234474-59163300-5ee4-46ac-9126-052074001be9.png)

Reply:
👨🏽‍💻  I'll make the proposed changes right away 😃 . Thanks a billion for this insightful review @Davidosky007  🤓 .

Reply:
👨🏽‍💻  I'll make the proposed changes right away 😃 . Thanks a billion for this insightful review @Davidosky007  🤓 .

Reply:
👨🏽‍💻  I'll make the proposed changes right away 😃 .

> Thanks a billion for this insightful review @Davidosky007  🤓 .

Reply:
Actioned ✅  .On to requesting for another review.

> HappyCoding/>